Gemfile

```diff
@@ -61,6 +61,8 @@ gem 'web-console', '~> 2.0', group: :development
 # test coverage in static code analysis GUI
 gem 'codeclimate-test-reporter', group: :test, require: false
 
+gem 'sidekiq'
```

Comment:
in keeping with this gemfile practice, there should be a comment indicating why we are including this gem.